README.md

```diff
@@ -19,18 +19,48 @@ You can install the latest version with WinGet:
 winget install Microsoft.Edit
 ```
 
-### Notes to Package Maintainers
-
-The canonical executable name is "edit" and the alternative name is "msedit".
-
-We're aware of the potential conflict of "edit" with existing commands and as such recommend naming packages and executables "msedit".
-Names such as "ms-edit" should be avoided.
-Assigning an "edit" alias is recommended if possible.
-
 ## Build Instructions
 
 * [Install Rust](https://www.rust-lang.org/tools/install)
 * Install the nightly toolchain: `rustup install nightly`
   * Alternatively, set the environment variable `RUSTC_BOOTSTRAP=1`
 * Clone the repository
 * For a release build, run: `cargo build --config .cargo/release.toml --release`
+
+## Notes to Package Maintainers
+
+### Package Naming
+
+The canonical executable name is "edit" and the alternative name is "msedit".
+We're aware of the potential conflict of "edit" with existing commands and recommend alternatively naming packages and executables "msedit".
+Names such as "ms-edit" should be avoided.
+Assigning an "edit" alias is recommended, if possible.
+
+### ICU library name (SONAME)
+
+This project _optionally_ depends on the ICU library for its Search and Replace functionality.
+By default, the project will look for a SONAME without version suffix:
+* Windows: `icuuc.dll`
+* macOS: `libicuuc.dylib`
```

Comment:
I think macOS only has `libicucore`, and not `uc`/`i18n`... hmm

Reply:
I think it also exports all of them as C++ symbols, i.e. with underscore prefix.